lib/bamboo/adapters/smtp_adapter.ex

```diff
@@ -161,6 +168,45 @@ defmodule Bamboo.SMTPAdapter do
     |> add_smtp_line(text_body)
   end
 
+  defp add_multipart_mixed_split(body, delimiter), do: "#{body}\r\n--#{delimiter}\r\n"
+
+  defp add_attachment_header(body, attachment) do
+    << random :: size(32) >> = :crypto.strong_rand_bytes(4)
+    "#{body}Content-Type: attachment.content_type; name=\"#{attachment.filename}\"\r\n" <>
```

Comment:
What do you think of using the `add_smtp_line` function instead of doing the concatenation yourself?

```elixir
body
|> add_smtp_line("Content-Type: attachment.content_type; name=\"#{attachment.filename}\"")
|> add_smtp_line("Content-Disposition: attachment; filename=\"#{attachment.filename}\")
# ....
```